backends/pass.go

```diff
@@ -56,7 +56,7 @@ func (p *Pass) ListSecrets(inputPath string) (secrets []string, err error) {
 		return nil, fmt.Errorf("failed to retrieve gopass tree: %s", err)
 	}
 
-	l := t.List(1)
+	l := t.List(100)
```

Comment:
How is that related to the file provider?

Reply:
Looks like `0` do the trick.